### DISCUSSION TITLE
Make it easier to add custom taxonomies on-the-fly from within Pod Edit for a Post Type (built-in taxonomies, just click "Add New Taxonomy" for inline form)

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6606] [Ideas]
Make it easier to add custom taxonomies on-the-fly from within Pod Edit for a Post Type (built-in taxonomies, just click "Add New Taxonomy" for inline form)

**Comment by JoryHogeveen** [comment]
@sc0ttkclark Do you have a mockup of what you mean?
